Description: Removing jarfiles which contain MacOSX-related classes, build
fails looking for missing classes. This patch fixes it.
Author: Gabriele Giacone <1o5g4r8o@gmail.com>

Index: sweethome3d-4.4/src/com/eteks/sweethome3d/tools/OperatingSystem.java
===================================================================
--- sweethome3d-4.4.orig/src/com/eteks/sweethome3d/tools/OperatingSystem.java	2014-09-16 21:58:40.779163615 +0200
+++ sweethome3d-4.4/src/com/eteks/sweethome3d/tools/OperatingSystem.java	2014-09-16 21:58:40.775163615 +0200
@@ -33,7 +33,7 @@
 import java.util.TimerTask;
 import java.util.UUID;
 
-import com.apple.eio.FileManager;
+//import com.apple.eio.FileManager;
 import com.eteks.sweethome3d.model.Home;
 
 /**
@@ -423,9 +423,10 @@
    */
   public static File getDefaultApplicationFolder() throws IOException {
     File userApplicationFolder; 
-    if (isMacOSX()) {
+/*    if (isMacOSX()) {
       userApplicationFolder = new File(MacOSXFileManager.getApplicationSupportFolder());
-    } else if (isWindows()) {
+    } else if (isWindows()) { */
+    if (isWindows()) {
       userApplicationFolder = new File(System.getProperty("user.home"), "Application Data");
       // If user Application Data directory doesn't exist, use user home
       if (!userApplicationFolder.exists()) {
@@ -446,10 +447,10 @@
    * This class requires some classes of <code>com.apple.eio</code> package  
    * to compile.
    */
-  private static class MacOSXFileManager {
+/*  private static class MacOSXFileManager {
     public static String getApplicationSupportFolder() throws IOException {
       // Find application support folder (0x61737570) for user domain (-32763)
       return FileManager.findFolder((short)-32763, 0x61737570);
     }
-  }
+  } */
 }
Index: sweethome3d-4.4/src/com/eteks/sweethome3d/SweetHome3D.java
===================================================================
--- sweethome3d-4.4.orig/src/com/eteks/sweethome3d/SweetHome3D.java	2014-09-16 21:58:40.779163615 +0200
+++ sweethome3d-4.4/src/com/eteks/sweethome3d/SweetHome3D.java	2014-09-16 21:58:40.775163615 +0200
@@ -466,7 +466,6 @@
     }
     if (OperatingSystem.isMacOSX()) {
       // Bind to application menu at last
-      MacOSXConfiguration.bindToApplicationMenu(this);
     }
 
     // Run everything else in Event Dispatch Thread
Index: sweethome3d-4.4/src/com/eteks/sweethome3d/HomeFramePane.java
===================================================================
--- sweethome3d-4.4.orig/src/com/eteks/sweethome3d/HomeFramePane.java	2014-09-16 21:58:40.779163615 +0200
+++ sweethome3d-4.4/src/com/eteks/sweethome3d/HomeFramePane.java	2014-09-16 21:58:40.775163615 +0200
@@ -123,7 +123,7 @@
       homeFrame.setIconImage(frameImages.get(0));
     }
     if (OperatingSystem.isMacOSXLionOrSuperior()) {
-      MacOSXConfiguration.installToolBar(this);
+//      MacOSXConfiguration.installToolBar(this);
     }
     updateFrameTitle(homeFrame, this.home, this.application);
     // Change component orientation
